spark-shell/src/main/scala/org/apache/mahout/sparkbindings/shell/MahoutSparkILoop.scala

```diff
@@ -48,55 +77,81 @@ class MahoutSparkILoop extends SparkILoop {
 
     conf.set("spark.executor.memory", "1g")
 
-    sparkContext = mahoutSparkContext(
+    _interp.sparkContext = mahoutSparkContext(
       masterUrl = master,
       appName = "Mahout Spark Shell",
       customJars = jars,
       sparkConf = conf
     )
 
```

Comment:
ok what i meant is that this should really be saved as MahoutDistributed context :

    private val sdc = mahoutSparkContext(...) 

which is already of type SparkDistributedContext (mahout type), right?

and then use to initialize `sdc = this.sdc` and `sc = this.sdc.sc` later... We still seem to be crating DistributedSparkContext twice here, which is probably not a big deal but feels a bit hacky...